src/error.rs

```diff
@@ -45,4 +45,10 @@ impl From<goblin::error::Error> for Error {
     }
 }
 
+impl From<anyhow::Error> for Error {
+    fn from(err: anyhow::Error) -> Error {
+        Error::Casr(err.chain().rev().map(|s| s.to_string() + ". ").collect())
```

Comment:
Why do we need this? Why do we change anyhow error to Casr?

Reply:
For convenient use `?`:
![image](https://user-images.githubusercontent.com/60780909/211582484-32bdecc6-647f-4ccc-8295-d8508bb39cb1.png)